src/function_space_operators.jl

```diff
@@ -0,0 +1,49 @@
+
+"""
+    GlaubitzNordströmÖffner2023()
+
+Function space SBP (FSBP) operators given in
+- Glaubitz, Nordström, Öffner (2023)
+  Summation-by-parts operators for general function spaces.
+  SIAM Journal on Numerical Analysis 61, 2, pp. 733-754.
+
+See also
+- Glaubitz, Nordström, Öffner (2024)
+  An optimization-based construction procedure for function space based
+  summation-by-parts operators on arbitrary grids.
+  arXiv, arXiv:2405.08770v1.
+"""
+struct GlaubitzNordströmÖffner2023 <: SourceOfCoefficients end
+
+function Base.show(io::IO, source::GlaubitzNordströmÖffner2023)
+  if get(io, :compact, false)
+    summary(io, source)
+  else
+      print(io,
+          "Glaubitz, Nordström, Öffner (2023) \n",
+          "  Summation-by-parts operators for general function spaces \n",
+          "  SIAM Journal on Numerical Analysis 61, 2, pp. 733-754. \n",
+          "See also \n",
+          "  Glaubitz, Nordström, Öffner (2024) \n",
+          "  An optimization-based construction procedure for function \n",
+          "    space based summation-by-parts operators on arbitrary grids \n",
+          "  arXiv, arXiv:2405.08770v1.")
+  end
+end
+
+"""
+    FunctionSpaceOperator(basis_functions, x_min, x_max, nodes, source; accuracy_order = 0)
+
+Construct a `MatrixDerivativeOperator` that represents a derivative operator in a function space.
+"""
+function FunctionSpaceOperator(basis_functions,
```

Comment:
A capitalized name suggests that it's a constructor returning something of type `FunctionSpaceOperator`, which is not the case. Maybe switch to `function_space_operator` or so? And document that it's not necessarily returning a `MatrixDerivativeOperator` as discussed earlier?

Reply:
Fixed in [a2ef507](https://github.com/ranocha/SummationByPartsOperators.jl/pull/268/commits/a2ef507933aa4964d4264d1a86427f0242d519b9).